docs/tasks/administer-cluster/reserve-compute-resources.md

```diff
@@ -40,46 +60,80 @@ on each node.
 ---------------------------
 ```
 
+<!--
 `Allocatable` on a Kubernetes node is defined as the amount of compute resources
 that are available for pods. The scheduler does not over-subscribe
 `Allocatable`. `CPU`, `memory` and `storage` are supported as of now.
+-->
+Kubernetes 节点上的 `Allocatable` 定义了 pod 可用计算资源量。调度器不会超额申请  `Allocatable`。目前支持 `CPU`, `memory` 和 `storage` 这几个参数。
 
+<!--
 Node Allocatable is exposed as part of `v1.Node` object in the API and as part
 of `kubectl describe node` in the CLI.
+-->
+Node Allocatable 暴露为 API 中 `v1.Node` 对象的一部分，也是 CLI 中 `kubectl describe node` 的一部分。
 
+<!--
 Resources can be reserved for two categories of system daemons in the `kubelet`.
+-->
+在 `kubelet` 中，可以为两类系统守护进程预留资源。
 
+<!--
 ### Enabling QoS and Pod level cgroups
+-->
+### 启用 QoS 和 Pod 层级的 cgroups
 
+<!--
 To properly enforce node allocatable constraints on the node, you must
 enable the new cgroup hierarchy via the `--cgroups-per-qos` flag.  This flag is
 enabled by default.  When enabled, the `kubelet` will parent all end-user pods
 under a cgroup hierarchy managed by the `kubelet`.
+-->
+为了恰当的在节点范围实施 node allocatable，您必须通过 `--cgroups-per-qos` 标志启用新的 cgroup 层次结构。这个标识时默认启用的。启用后，`kubelet` 将在其管理的 cgroup 层次结构中创建所有终端用户的 pod。
 
+<!--
 ### Configuring a cgroup driver
+-->
+### 配置 cgroup 驱动
 
+<!--
 The `kubelet` supports manipulation of the cgroup hierarchy on
 the host using a cgroup driver. The driver is configured via the
 `--cgroup-driver` flag.
+-->
+`kubelet` 支持在主机上使用 cgroup 驱动操作 cgroup 层次结构。驱动通过 `--cgroup-driver` 标志配置。
 
+<!--
 The supported values are the following:
 
 * `cgroupfs` is the default driver that performs direct manipulation of the
-cgroup filesystem on the host in order to manage cgroup sandboxes.
+  cgroup filesystem on the host in order to manage cgroup sandboxes.
 * `systemd` is an alternative driver that manages cgroup sandboxes using
-transient slices for resources that are supported by that init system.
+  transient slices for resources that are supported by that init system.
+  -->
+支持的参数值如下：
+
+* `cgroupfs` 是默认的驱动，在主机上直接操作 cgroup 文件系统以对 cgroup 沙箱进行管理。
+* `systemd` 是可替代的驱动，使用资源的瞬时切片管理 cgroup 沙箱，这需要 init 系统支持。
 
+<!--
 Depending on the configuration of the associated container runtime,
 operators may have to choose a particular cgroup driver to ensure
 proper system behavior.  For example, if operators use the `systemd`
 cgroup driver provided by the `docker` runtime, the `kubelet` must
 be configured to use the `systemd` cgroup driver.
+-->
+取决于相关容器运行时的配置，操作员可能需要选择一个特定的 cgroup 驱动来保证系统正常运行。例如如果操作员使用 `docker` 运行时提供的 cgroup 驱动时，必须配置 `kubelet` 使用 `systemd` cgroup 驱动。
```

Comment:
“运行时” 容易引起疑议 翻译成 “运行环境” 较容易理解

Reply:
例如 -> 例如,

Reply:
“运行时” is commonly used in software territory, e.g. "JAVA 运行时，.net 运行时". But considering the context, it may cause come misunderstanding here. So I'd like to append the English after it like this: 容器运行时（container runtime）.